swift/Samples/TabBarContainer/DemoApp/Bar/BarScreen.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Workflow
+import WorkflowUI
+
+
+struct BarScreen: Screen {
+    let title: String
+    let backgroundColor: UIColor
+    let viewTapped: () -> Void
+
+    func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription {
+        return BarScreenViewController.description(for: self, environment: environment)
+    }
+}
+
+
+fileprivate final class BarScreenViewController: ScreenViewController<BarScreen> {
+    
+    private lazy var titleLabel: UILabel = .init()
+    private lazy var tapGestureRecognizer: UITapGestureRecognizer = .init()
+    
+    required init(screen: BarScreen, environment: ViewEnvironment) {
+        super.init(screen: screen, environment: environment)
+        
+        update(with: screen)
```

Comment:
From what i understand screenDidChange() will now be called right after init so this update function call is not needed anymore.